lighthouse-core/test/gather/computed/critical-request-chains-test.js

```diff
@@ -7,9 +7,9 @@
 
 /* eslint-env mocha */
 
-const GathererClass = require('../../../gather/computed/critical-request-chains');
 const assert = require('assert');
-const Gatherer = new GathererClass();
+
+const Runner = require('../../../runner.js');
```

Comment:
well that's annoying, can't we just pass an obj like `{requestNetworkRecords: () =>...}` to `compute_` instead?